dova/www/templates/dashboard.html

```diff
@@ -1,8 +1,43 @@
 <ion-view view-title="Dashboard" hide-back-button="true" id="page10" ng-init="loadEnrollments()" cache-view="false">
-  <ion-content padding="true" class="has-header">
-    <ion-list id="dashboard-list9" ng-repeat="enrollment in enrollments">
+  <ion-content padding="true" class="has-header manual-remove-top-padding" scroll="false">
 
-      <ion-item class="item-thumbnail-left item-icon-right  course" id="dashboard-list-item45" ng-click="toAssignments(enrollment.id)" nav-direction="forward">
+    <ion-scroll class="top-container">
```

Comment:
using an ion-scroll allows me to put the content of the page all in that container, and have it be scrollable, so I can add a form to the bottom of the page, aka the delete button.